recipes/ixwebsocket/all/test_package/SimpleTests.cpp

```diff
@@ -0,0 +1,80 @@
+#include <iostream>
```

Comment:
Try something simpler, which don't requires networking. We don't want an external factor when running CI jobs. Also, test_package servers to validate the package, not to exercise the project, thus it can be really simple.

Reply:
Oh, I thought the point was to test the package. What's considered minimal then? Include, but don't actually run anything?

Reply:
Yes, the idea is testing the package, but the Conan package itself. Usually using an unit test from author's project is not a bad idea, but sometimes they more complex, like this one. You can try some test from `test/IXUrlParserTest.cpp`, the parser for instance. You will be linking the library, which is required to check any linkage error, and won't consume network traffic to run.